nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/admin/ListHistoricRequestTraces.java

```diff
@@ -101,7 +101,7 @@ public void execute(AdminCommandContext context) {
         RequestTracingExecutionOptions executionOptions = service.getExecutionOptions();
         if (!executionOptions.isHistoricalTraceEnabled()) {
             actionReport.setMessage("Request Tracing Historical Trace is not enabled!");
-            actionReport.setActionExitCode(ActionReport.ExitCode.FAILURE);
+            actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
```

Comment:
An error is thrown in the logs whenever the request tracing page is loaded because the historical request tracing is disabled. The command should return with a success, saying it's disabled